package.json

```diff
@@ -41,8 +41,11 @@
   "scripts": {
     "make": "npm run lint && npm run build && npm run test",
     "lint": "tslint --fix --config tslint.json 'src/**/*'",
+    "lint:ci": "tslint -t checkstyle -o reports/checkstyle.xml --config tslint.json 'src/**/*'",
     "test": "nyc mocha",
+    "test:unit": "nyc --reporter=text-summary --reporter=cobertura --report-dir=reports  mocha",
     "build": "tsc",
+    "tsc": "tsc",
```

Comment:
Perhaps this command should actually be called 'compile'

Reply:
ok to rename to "compile" to clarify